spm12/metadata/get_metadata_val_classic.m

```diff
@@ -123,8 +123,16 @@
         end
         
     case 'EchoTime' % [ms]
-        % Valid for all vendors
+        % Valid for all vendors:
+        % try EchoTime first
         [nFieldFound, fieldList] = find_field_name(mstruc, 'EchoTime', 'caseSens','sensitive','matchType','exact');
+        % TerraX acquisition with new Dicoms chosen:
+        % If EchoTime is not found than first search for EffectiveEchoTime
+        % with the same arguments to find_field_name as in EchoTime
+        if ~nFieldFound
+            [nFieldFound, fieldList] = find_field_name(mstruc,'EffectiveEchoTime', 'caseSens','sensitive','matchType','exact');
```

Comment:
It might be a good idea to make this a `'partial'` match for `'EchoTime'` rather than an `'exact'` match for `'EffectiveEchoTime'`. That way we might make ourselves robust to other changes in the future.

Reply:
Who do you mean by "they"?

Reply:
Blergh... I have no idea why they would specify `EchoTime1` and `EchoTime2`, but since they do, you are right that we should rely on exact matching.

Please note that `EffectiveEchoTime` is from our DICOM converter, not any BIDS DICOM converter, so it should not be expected to be a BIDS field.